core/templates/pages/welcome-guide-page/welcome-guide-page.directive.html

```diff
@@ -0,0 +1,79 @@
+<div style="height: auto; background-color: #FFFFFF;" class="row content">
+  <div  class="col-lg-7">
+    <div class="oppia-welcome-guide-header">
+      <h1 class="oppia-welcome-guide-h1" translate="I18N_WELCOME_GUIDE_TITLE"> </h1>
+      <p class = "oppia-welcome-guide-p" translate="I18N_WELCOME_GUIDE_PARAGRAH_1">
+      </p>
+
+    </div>
+  </div>
+  <div class="oppia-welcome-guide-img col-lg-5">
+    <picture>
+      <source type="image/webp" ng-srcset="<[$ctrl.headerImgUrlWebp]>">
+      <source type="image/jpeg" ng-srcset="<[$ctrl.headerImgUrl]>">
+      <img ng-src="<[$ctrl.headerImgUrl]>" alt="Learning with Oppia">
+    </picture>
+  </div>
+</div>
+
+<div style=" background-color: #E5E5E5" class="row content">
+  <div class="col-lg-6">
+    <div class="oppia-welcome-guide-second-section">
+      <div style = "padding-bottom: 40px;">
+        <h1 class="oppia-welcome-guide-h1" translate="I18N_WELCOME_PAGE_COLLECTIONS_TITLE"> </h1>
+        <div  class="oppia-welcome-guide-p" translate="I18N_WELCOME_PAGE_COLLECTIONS_PARAGRAPH">
+
+        </div>
+      </div>
+      <div style = "padding-bottom: 40px;">
+        <h1 class="oppia-welcome-guide-h1" translate="I18N_WELCOME_PAGE_EXPLORATION_TITLE"> </h1>
+        <div class="oppia-welcome-guide-p" translate="I18N_WELCOME_PAGE_EXPLORATION_PARAGRAPH">
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="col-lg-6 oppia-welcome-guide-fractions-image">
+    <picture>
+      <source type="image/webp" ng-srcset="<[$ctrl.fractionImgUrlWebp]>">
+      <source type="image/png" ng-srcset="<[$ctrl.fractionImgUrl]>">
+      <img class = "oppia-welcome-guide-fractions-img" ng-src="<[$ctrl.fractionImgUrl]>" alt="fraction pic">
+    </picture>
+  </div>
+</div>
+
+<div style="background-color: #FFFFFF" class="row content">
+  <div class="col-lg-12">
+    <div class="oppia-welcome-guide-third-section">
+      <h1 class="oppia-welcome-guide-h1" translate="I18N_WELCOME_GUIDE_HOW_TO_TITLE"> </h1>
+      <div class="oppia-welcome-guide-block">
+        <p translate="I18N_WELCOME_GUIDE_PARAGRAH_2"></p>
+        <p translate="I18N_WELCOME_GUIDE_PARAGRAH_3"></p>
+        <p translate="I18N_WELCOME_GUIDE_PARAGRAH_4"></p>
+      </div>
+    </div>
+  </div>
+
+  <div class="oppia-welcome-guide-exploration-image col-lg-6">
+    <picture>
+      <source type="image/webp" ng-srcset="<[$ctrl.browserImgUrlWebp]>">
+      <source type="image/jpeg" ng-srcset="<[$ctrl.browserImgUrl]>">
+      <img style="width: 625px;" ng-src="<[$ctrl.browserImgUrl]>" alt="Screenshot of Oppia explorations page">
+    </picture>
+  </div>
+  <div class="oppia-welcome-guide-browser-image col-lg-6">
+    <span class="oppia-welcome-guide-blue">1</span> <p class="oppia-welcome-guide-numbers" translate="I18N_WELCOME_GUIDE_BROWSER-BULLET_1"> </p>
```

Comment:
Can we use ng-repeat here?

Reply:
could we possibly table this for a future pr?

Reply:
fixed